crates/bifrost/src/providers/replicated_loglet/network.rs

```diff
@@ -172,33 +172,15 @@ impl RequestPump {
             return;
         }
 
-        match loglet.find_tail().await {
-            Ok(tail) => {
-                let sequencer_state = SequencerState {
-                    header: CommonResponseHeader {
-                        known_global_tail: Some(tail.offset()),
-                        sealed: Some(tail.is_sealed()),
-                        status: SequencerStatus::Ok,
-                    },
-                };
-                let _ = reciprocal.prepare(sequencer_state).try_send();
-            }
-            Err(err) => {
-                // Well, we couldn't finish FindTail. Very likely we failed in CheckSeal. In all
```

Comment:
CheckSeal happens periodically, right?

Reply:
Yes. It's run by a periodic find tail, albeit this solution is still a shortcut and will likely cause "restatectl logs reconfigure" to crash. But that is not a blocker at the moment.

Reply:
What's the problem for `restatectl logs reconfigure` now?